sql/migrations.go

```diff
@@ -89,11 +89,6 @@ func (m *sqlMigration) Order() int {
 	return m.order
 }
 
-func (sqlMigration) Rollback() error {
-	// handled by the DB itself
-	return nil
-}
```

Comment:
This method is not always empty and in my opinion should stay part of the interface for migrations.

Reply:
Reverted this part of the PR